src/backend/distributed/metadata/node_metadata.c

```diff
@@ -380,6 +380,9 @@ master_set_node_property(PG_FUNCTION_ARGS)
  * - All dependencies (e.g., types, schemas)
  * - Reference tables, because they are needed to handle queries efficiently.
  * - Distributed functions
+ *
+ * Note that we do not create the distributed dependencies since all the dependencies
```

Comment:
I mistyped the comment here, fixing it:

Note that we do not create the distributed dependencies **on the coordinator** since all the dependencies should be present in the coordinator already